command/operator_migrate.go

```diff
@@ -209,7 +223,7 @@ func (c *OperatorMigrateCommand) migrate(config *migratorConfig) error {
 
 	doneCh := make(chan error)
 	go func() {
-		doneCh <- c.migrateAll(ctx, from, to)
+		doneCh <- c.migrateAll(ctx, from, to, c.flagParallel)
```

Comment:
You'll probably need `int(c.flagParallel)` now if we move to `parallel` being an unsigned int.